src/Entity/Group/Relation.php

```diff
@@ -85,6 +85,13 @@ public function getId(): ?string
         return $this->id;
     }
 
+    public function setId(string $id): self
```

Comment:
Why are these (and other) setId() methods added?

Reply:
It's needed for the alice bundle to hydrate the property's. There were some other possibilities but this looked the nicest, and everyone understands what's they see. (where people might need to look a second and third time if we for example used `__call()`.

Reply:
In that case, a comment should be added (here and in all other instances) indicating the intented use and stating that this function should not be used any further, since Doctrine relies upon this value to be readonly in practice.